src/cfnlint/data/CloudSpecs/ap-northeast-3.json

```diff
@@ -16580,44 +16580,29 @@
       }
     },
     "AWS::EC2::VPCEndpoint": {
-      "Attributes": {
-        "CreationTimestamp": {
-          "PrimitiveType": "String"
-        },
-        "DnsEntries": {
-          "PrimitiveItemType": "String",
-          "Type": "List"
-        },
-        "NetworkInterfaceIds": {
-          "PrimitiveItemType": "String",
-          "Type": "List"
-        }
-      },
       "Documentation": "http://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-ec2-vpcendpoint.html",
       "Properties": {
-        "PolicyDocument": {
-          "Documentation": "http://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-ec2-vpcendpoint.html#cfn-ec2-vpcendpoint-policydocument",
-          "PrimitiveType": "Json",
+        "IsPrivateDnsEnabled": {
```

Comment:
Looks like there might be an issue with a spec patch here. I'll dig in tomorrow.